app/main.py

```diff
@@ -112,3 +118,53 @@ async def knowledge_panel(
                 logging.exception()
 
         return {"knowledge_panels": panels}
+
+
+templates = Jinja2Templates(directory="template")
+
+
+@app.get("/render-to-html", tags=["Render to HTML"], response_class=HTMLResponse)
+async def render_html(
```

Comment:
I think we should have a parameter to say if we want a full html or just the "entry" div (for inclusion on an existing page).

Reply:
I just removed Html and body tags because even for full HTML rendering HTML tags aren't essential.